torchrl/objectives/a2c.py

```diff
@@ -146,9 +147,10 @@ class A2CLoss(LossModule):
         ...     observation = torch.randn(*batch, n_obs),
         ...     action = spec.rand(batch),
         ...     next_done = torch.zeros(*batch, 1, dtype=torch.bool),
-        ...     next_reward = torch.randn(*batch, 1))
+        ...     next_reward = torch.randn(*batch, 1),
+        ...     next_observation = torch.randn(*batch, n_obs))
         >>> loss_val
-        (tensor(1.7593, grad_fn=<MeanBackward0>), tensor(0.2344, grad_fn=<MeanBackward0>), tensor(1.5480), tensor(-0.0155, grad_fn=<MulBackward0>))
+        (tensor(4.3483, grad_fn=<MeanBackward0>), tensor(1.4114, grad_fn=<MeanBackward0>), tensor(2.5165), tensor(-0.0252, grad_fn=<MulBackward0>))
```

Comment:
maybe we don't want to print these values (such that we don't pollute PRs with diffs that aren't relevant?)

Reply:
Good point, I will just remove it.